layouts/_default/baseof.html

```diff
@@ -7,14 +7,18 @@
     {{ hugo.Generator }}
     <link rel="icon" href="{{ .Site.Params.favicon | absURL }}" type="image/x-icon">
     {{ $style := resources.Get "presidium.scss" | resources.ToCSS | resources.Minify | resources.Fingerprint }}
+    <link rel="preload" as="style" href="{{ $style.Permalink }}" as="style">
     <link rel="stylesheet" href="{{ $style.RelPermalink }}">
     {{ if resources.Get "index.scss" }}
       {{ with resources.Get "index.scss" | resources.ToCSS | resources.Minify | resources.Fingerprint }}
         <link rel="stylesheet" href="{{ .RelPermalink }}">
       {{ end }}
     {{ end }}
-    <script src="{{ "assets/scripts/jquery.js" | absURL }}"></script>
-    <script src="{{ "assets/scripts/bootstrap.js" | absURL }}"></script>
+
+    {{ if $.Site.Params.enterprise_enabled }}
+      <link rel="preload" src="/presidium-enterprise.js" as="script">
```

Comment:
Seeing as we're including the script partial on this page I think we can just defer here and it will be executed once HTML has parsed. Or am I missing a reason for preloading first?

Reply:
The "preload" tells the browser this script is a higher priority and should be loaded as early as possible. [from docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload). 

> Specifying resources your page will need very soon, which you want to start loading early in the page lifecycle, before browsers' main rendering machinery kicks in. This ensures they are available earlier and are less likely to block the page's render, improving performance. 

With defer the browser can still load the script in whatever order it wants but it preserves the execution order.

The idea is to reduce the time it takes for enterprise features to render. It doesn't make a huge difference in performance but I noticed that the lighthouse scores were much more consistent when I used preload. Without preload, the scores fluctuate between 40 and 70, whereas with preload, it consistently scores above 62.